lib/_stream_writable.js

```diff
@@ -688,6 +697,7 @@ function endWritable(stream, state, cb) {
   }
   state.ended = true;
   stream.writable = false;
+  debug('endWritable', state.ended);
```

Comment:
the `state.ended` will always be true so I'm not sure how useful this log is.

Reply:
Get it!